src/__tests__/queries.test.ts

```diff
@@ -1,66 +1,54 @@
 import { Akord } from "../index";
 import { email, password } from './data/test-credentials';
-import { vaults, fileId, message, publicVaultId, privateVaultId } from './data/content';
+import { vaults, fileId, message, publicVaultId, privateVaultId, parentId } from './data/content';
 import { NodeJs } from "../types/file";
 
-let clientWithAkordApi: Akord;
-let clientWithArweaveApi: Akord;
-let clientWithoutWallet: Akord;
+let privateClient: Akord;
+let publicClient: Akord;
 
 jest.setTimeout(3000000);
 
 describe("Testing querying directly from permaweb", () => {
   beforeAll(async () => {
     const { jwtToken, wallet } = await Akord.auth.signIn(email, password);
-    clientWithAkordApi = new Akord(wallet, jwtToken, { wallet: <any>"Akord" });
-    clientWithArweaveApi = new Akord(wallet, undefined, { wallet: <any>"Arweave" });
-    clientWithoutWallet = new Akord(undefined, undefined, { wallet: <any>"Akord" });
-
+    privateClient = new Akord(wallet, jwtToken);
+    publicClient = new Akord();
   });
 
-  it("Query all vaults from Akord API", async () => {
-    const result = await clientWithAkordApi.vault.list();
-    expect(result).toEqual(vaults);
-  });
+  // it("Query all vaults from Akord API", async () => {
```

Comment:
not working?

Reply:
should work ok, I've commented it out, because it will give different results depending on the tester ;)

Reply:
I will prepare some test user & data in the next PR :)